Tests/Benchmarks/SolenoidFieldBenchmark.cpp

```diff
@@ -92,4 +112,63 @@ int main(int argc, char* argv[]) {
   const auto map_rand_result = Acts::Test::microBenchmark(
       [&] { return bFieldMap.getField(genPos()); }, iters_map);
   std::cout << map_rand_result << std::endl;
+  csv("interp_nocache_random", map_rand_result);
+
+  {
+    std::cout << "Benchmarking cached interpolated field lookup: "
+              << std::flush;
+    Cache cache{mctx};
+    const auto map_cached_result_cache = Acts::Test::microBenchmark(
+        [&] { return bFieldMap.getField(fixedPos, cache); }, iters_map);
+    std::cout << map_cached_result_cache << std::endl;
+    csv("interp_cache_fixed", map_cached_result_cache);
+  }
+
+  // - The second benchmark generates random positions, so it is biased by the
+  //   cost of random position generation and has unrealistically bad cache
+  //   locality, but provides an upper bound of field lookup performance.
```

Comment:
I'm not sure if this comment should stay or not. It feels out of place in the middle of the surrounding documentation vacuum. If you're not going to add any comments to the others, maybe let's remove this one as well?

Reply:
I can remove it, yeah.

Reply:
I did the opposite and added additional comments to the new benchmarks. Let me know if you're happy with them.